components/esp_lvgl_port/docs/performance.md

```diff
@@ -0,0 +1,125 @@
+# LCD & LVGL Performance
+
+This document provides steps, how to set up your LCD and LVGL port for the best performance and comparing the difference settings. All settings and measurements are valid for Espressif's chips.
+
+## Process of the performance measuring
+
+There are three main processes for measurement performance on LCD:
+
+1. Measure time, how long it takes to refresh the whole screen.
+2. Use `lv_demo_benchmark()` to measure FPS (weighted FPS).
+3. Use `lv_demo_music()` to measure FPS (average FPS).
+
+## The settings on ESP32 chips which have impact for LCD and LVGL performance
+
+These options and settings have impact for LCD performance (FPS). Some options have only small difference of FPS (e.g. 1 FPS), and some of them are better. Usually it depends on application, resources and size of screen. Also the HW interface and clock speed is really important (not included in this sheet).
+
+### LVGL Buffer size
+
+* The size of **LVGL buffer** and **double buffering enables** has big impact on performance. 
+* The best is use the same size of the buffer, like size of the LCD in pixels and enable double buffering. 
+* If there is not enough RAM inside the CPU, you can use PSRAM (DMA is not available).
+
+### Compiler optimization level
+
+Recommended level is "Performance" for good results. The "Performance" setting causes the compiled code to be larger and faster, but will be easier to correlate code addresses to source file lines.
```

Comment:
'easier to correlate code addresses to source file lines' isn't it the other way around?